packages/web/src/scss/components/TextField/_TextField.scss

```diff
@@ -46,16 +46,10 @@
     width: 100%;
 }
 
-.TextField:hover > .TextField__input,
-.TextField__input:hover {
-    border-color: theme.$input-border-color-hover;
-}
-
-.TextField:focus-within,
-.TextField:focus-within:hover,
-.TextField > .TextField__input:focus {
+.TextField .TextField__input:focus-visible {
```

Comment:
OK, so now it can be moved as under `.TextField__input` as `&:focus-visible`, right? Or is the higher specificity necessary?

Reply:
We need the higher specificity to override color variants like `error` or `warning`.